modules/ROOT/pages/notification-details.adoc

```diff
@@ -0,0 +1,1112 @@
+----
+
+[NOTE]
+====
+**Description of notification** +
+Databases and aliases with unescaped `.` are deprecated unless to indicate that they belong to a composite database. Names containing `.` should be escaped. (Name: foo.bar)
+====
+
+.Suggestions of improvement
+
+If not intended for a composite database, escape the name with the character ```.
+
+[source,cypher]
+----
+CREATE DATABASE `foo.bar`
+----
+======
+
+.A procedure has been deprecated or renamed
+======
+
+[source,cypher]
+----
+CALL unsupported.dbms.shutdown
+----
+
+[NOTE]
+====
+**Description of notification** +
+The query used a deprecated procedure. ('unsupported.dbms.shutdown' is no longer supported)
+====
+
+======
+
+.Using a deprecated runtime option
+======
+
+[source,cypher]
+----
+CYPHER runtime = interpreted MATCH (n) RETURN n
+----
+
+[NOTE]
+====
+**Description of notification** +
+The query used a deprecated runtime option. ('runtime=interpreted' is deprecated, please use 'runtime=slotted' instead)
+====
+
+.Suggestions of improvement
+
+Runtime `interpreted` is deprecated, use another runtime or remove the runtime option in order to use the default runtime.
+
+[source,cypher]
+----
+MATCH (n) RETURN n
+----
+======
+
+.Using the text-1.0 index provider when creating a text index
+======
+
+[source,cypher]
+----
+CREATE TEXT INDEX FOR (n:Label) ON (n.prop) OPTIONS {indexProvider : 'text-1.0'}
+----
+
+[NOTE]
+====
+**Description of notification** +
+The `text-1.0` provider for text indexes is deprecated and will be removed in a future version.Please use `text-2.0` instead.
+====
+
+.Suggestions of improvement
+
+Update the option `indexProvider` to have the value `text-2.0`.
+
+[source,cypher]
+----
+CREATE TEXT INDEX FOR (n:Label) ON (n.prop) OPTIONS {indexProvider : 'text-2.0'}
+----
+======
+
+[#_neo_clientnotification_request_deprecatedformat]
+=== Neo.ClientNotification.Request.DeprecatedFormat
+
+[options="header",cols="<1m,<3"]
+|===
+
+|Code
+|Neo.ClientNotification.Request.DeprecatedFormat
+
+|SeverityLevel
+|WARNING
+
+|Category
+|DEPRECATION
+
+|Title
+|The client made a request for a format which has been deprecated.
+|===
+
+=== Generic
+
+[#_neo_clientnotification_statement_subqueryvariableshadowing]
+=== Neo.ClientNotification.Statement.SubqueryVariableShadowing
+
+[options="header",cols="<1m,<3"]
+|===
+
+|Code
+|Neo.ClientNotification.Statement.SubqueryVariableShadowing
+
+|SeverityLevel
+|INFORMATION
+
+|Category
+|GENERIC
+
+|Title
+|Variable in subquery is shadowing a variable with the same name from the outer scope.
+|===
+
+.Shadowing of a variable from the outer scope
+======
+
+[source,cypher]
+----
+MATCH (n)
+CALL {
+  MATCH (n)--(m)
+  RETURN m
+}
+RETURN *
+----
+
+[NOTE]
+====
+**Description of notification** +
+Variable in subquery is shadowing a variable with the same name from the outer scope.If you want to use that variable instead, it must be imported into the subquery using importing WITH clause. (the shadowing variable is: n)
```

Comment:
```suggestion
Variable in subquery is shadowing a variable with the same name from the outer scope. If you want to use that variable instead, it must be imported into the subquery using importing WITH clause. (the shadowing variable is: n)
```

Reply:
So strange, these are just copied, but it seems like something happened with the space. I'll update